optuna/samplers/_tpe/multivariate_parzen_estimator.py

```diff
@@ -259,26 +259,29 @@ def _transform_from_uniform(
 
             assert isinstance(distribution, _DISTRIBUTION_CLASSES)
             if isinstance(distribution, distributions.UniformDistribution):
-                transformed[param_name] = samples
+                transformed[param_name] = np.array(samples)
```

Comment:
Isn't `samples` always an ndarray here given the type hints from the method signature?